evalml/pipelines/components/transformers/feature_selection/feature_selector.py

```diff
@@ -70,3 +70,29 @@ def fit_transform(self, X, y=None):
             pd.DataFrame: Transformed data.
         """
         return self.fit(X, y).transform(X, y)
+
+    def _handle_partial_dependence_fast_mode(self, X, pipeline_parameters):
+        """Updates pipeline parameters to not drop any features based off of feature importance.
+
+        Note:
+            This is needed, because fast mode refits cloned pipelines on single columns,
+            so categorical columns that have one-hot encoding applied may lose some
+            of their encoded columns with the default parameters.
+
+        Args:
+            X (pd.DataFrame): Holdout data being used for partial dependence calculations.
+            pipeline_parameters (dict): Pipeline parameters that will be used to create the pipelines
+                used in partial dependence fast mode.
+
+        Return:
+            pipeline_parameters (dict): Pipeline parameters updated to allow the FeatureSelector component
+                to not drop any features. This is needed because the cloned pipeline that uses these features
+                will be fit on a single column, and we do not want to drop that column. We can use the original
+                pipeline to determine if that feature gets dropped or not.
```

Comment:
It feels odd and/or slightly redundant to mention the "this is needed" within the return statement. It might fit better and be easier to reference in the note section instead?

Reply:
Good call -- removed that section here and added another sentence above, so the note section now says:     

>  This is needed, because fast mode refits cloned pipelines on single columns,
>             so categorical columns that have one-hot encoding applied may lose some
>             of their encoded columns with the default parameters. Therefore, we update the
>             parameters here to not drop any columns, and use the original
>             pipeline to determine if that feature gets dropped or not.